site.mk

```diff
@@ -1,29 +1,29 @@
+GLUON_MULTIDOMAIN=1
+
+GLUON_FEATURES := \
+	respondd \
+	autoupdater \
+	web-wizard \
+	ebtables-limit-arp \
+	ebtables-filter-multicast \
+	ebtables-filter-ra-dhcp \
+	web-admin \
+	web-autoupdater \
+	web-network \
+	web-wifi-config \
+	mesh-batman-adv-15 \
+	mesh-vpn-fastd \
+	radvd \
+	status-page \
+	config-mode-domain-select \
+	ebtables-filter-arp-ffmuc \
+	ebtables-filter-multicast-ffmuc \
```

Comment:
this should be obsoleted by now by gluons own features.

Reply:
[This](https://github.com/freifunkMUC/gluon-packages/blob/master/gluon-ebtables-filter-multicast-ffmuc/files/lib/gluon/ebtables/110-mcast-allow-cjdns) makes me think, that this allows multicast for cjdns. Is that really supported in gluon out of the box?
Either way: The question stays: Should we keep (cjdns multicast support) of remove this entirely?